.github/workflows/verify-pr.yaml

```diff
@@ -24,9 +24,7 @@ jobs:
         run: npm run lint
 
       - name: Set up mdBook 📚
-        uses: ./.github/workflows/common/set-up-mdbook
```

Comment:
If you are removing this, then you can also remove `./.github/workflows/common/set-up-mdbook`

Reply:
I changed the common script instead. Running cargo install is slower, but more aligned with local usage, and this repo is not so time-sensitive.